src/home/index.html

```diff
@@ -52,5 +53,170 @@ <h1 class="heading">I am Mwafrika Glad to see you!</h1>
+          <div class="dot"></div>
+          <div class="gray">2015</div>
+        </div>
+        <p class="tonicDesc">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <div class="skills">
+          <p>html</p>
+          <p>css</p>
+          <p>javascript</p>
+        </div>
+        <div class="btn">
+          <button class="btn-tonic">See Project</button>
+        </div>
+      </section>
+      <section>
+        <h1 class="heading">About Myself</h1>
+        <p class="description">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <p class="subdescription">LET'S CONNECT</p>
+        <div class="social-container">
+          <ul>
+            <li class="social">
+              <a href="#">
+                <img src="../assert/images/twitter.png" alt="Twitter" />
+              </a>
+            </li>
+            <li class="social">
+              <a href="#">
+                <img src="../assert/images/Linkedin.png" alt="LinkedIn" />
+              </a>
+            </li>
+            <li class="social">
+              <a href="#">
+                <img src="../assert/images/m.png" alt="Medium" />
+              </a>
+            </li>
+            <li class="social">
+              <a href="#">
+                <img src="../assert/images/github.png" alt="Github" />
+              </a>
+            </li>
+            <li class="social">
+              <a href="#">
+                <img src="../assert/images/fingers.png" alt="Finger" />
+              </a>
+            </li>
+          </ul>
+        </div>
+        <div class="btn">
+          <button class="btn-tonic">See Project</button>
+        </div>
+        <div class="language-title">
+          <p>Language</p>
+          <img src="../assert/images/chevron-h.png" alt="language" />
+        </div>
+        <section class="langage-list">
+          <div class="lang">
+            <img src="../assert/images/js.png" alt="javascript" />
+            <p>Javascript</p>
+          </div>
+          <div class="lang">
+            <img src="../assert/images/css.png" alt="css" />
+            <p>CSS</p>
+          </div>
+          <div class="lang">
+            <img src="../assert/images/html.png" alt="html" />
+            <p>HTML</p>
+          </div>
+          <div class="language-title">
+            <p>Frameworks</p>
+            <img src="../assert/images/chevron-v.png" alt="chevron" />
+          </div>
+          <div class="line"></div>
+          <div class="language-title">
+            <p>Skills</p>
+            <img src="../assert/images/chevron-v.png" alt="chevron" />
+          </div>
+          <div class="line"></div>
+        </section>

```

Comment:
I think the technology categories ("Languages", "Frameworks" and "Skills") should be in a `ul` tag. Each of them in a `li` tag. 
The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` tag and each of them a `li` tag. 
Please make sure to use `ul` and `li` tag here 😊.

Reply:
All the implementation can be found on the screenshot that I added to the readme file.